app/_hub/kong-inc/request-validator/_index.md

```diff
@@ -34,6 +34,7 @@ params:
         The request body schema specification. One of `body_schema` or `parameter_schema`
         must be specified.
     - name: allowed_content_types
+      minimum_version: "3.1.x"
```

Comment:
I'm confused. Which release are all these changes going into, exactly?

Reply:
There are a couple of things

- content-type with parameter was introduced at v2.8 and > v3.1
- fixing breaking behavior at v2.8
- 3.0 is a gap(for some reason) that as same as version < v2.7  (I found the v3.0.x is showing the 2.8.x's description, it seems like the (minimum_version: "3.0.x"  and maximum_version: "3.0.x" does not work)

Reply:
Very weird. Luckily, we'll have this type of thing under control soon, as we're switching to generating these docs from plugin schemas in the near future :) Thanks for the fixes!

Reply:
LGTM. I made another small change.